vas2nets/tasks.py

```diff
@@ -0,0 +1,37 @@
+import csv
+import requests
+import pytz
+
+from django.conf import settings
+from django.utils.dateparse import parse_datetime
+from celery.task import Task
+
+from .models import VoiceCall
+
+
+class FetchVoiceData(Task):
+
+    def get_data(self, date):
+        url = "%s?report_date=%s" % (settings.V2N_VOICE_URL, date)
+
+        content = requests.get(url, stream=True)
+
+        return csv.DictReader(
+            [line.decode('utf-8') for line in content.iter_lines()])
```

Comment:
`iter_lines` has a `decode_unicode` parameter for just this purpose. Maybe we can use that for cleaner code?